### DISCUSSION TITLE
Simulation Parameter Selection

### DISCUSSION THREAD

**Discussion by LTMeyer** [PolymathicAI/the_well#11] [Q&A] [answered]
How did you select the parameters of the different simulations?

**Comment by rubenohana** [comment, marked as answer]
We had a minimum and maximum storage size we were looking at along with a target trajectory length to ensure that this data could be used for projects focusing on long-run stability of models. With these constraints, we wanted to ensure we had a sufficiently large dataset for training neural networks, so we worked with the domain experts to identify parameters and families of initial conditions that would produce interestingly different trajectories. We were especially interested in parameter values that would result in different physical behaviors. We also asked for a minimum of 10 different initial conditions for each physical parameter value or combination.

**Comment by wasabicoco** [comment]
We are working on the FDS fire dynamics model. Can we consider adding a fire dynamics model

**Reply by LTMeyer** [reply]
Hi @wasabicoco thank you for reaching out.
We detailed in the FAQ the contribution guidelines for external datasets https://github.com/PolymathicAI/the_well/discussions/20. It explains how to make a dataset compatible with the Well. Nonetheless, you would still have to host the data yourself. 
Please, let us if anything remains unclear with the instructions.
